chains/evm/contracts/offRamp/OffRamp.sol

```diff
@@ -115,6 +113,10 @@ contract OffRamp is ITypeAndVersion, Ownable2StepMsgSender {
 
   // DYNAMIC CONFIG
 
+  /// @notice Gas buffer to update state.
+  // Example, 5k for updating the state + 5k for the event and misc costs.
+  uint32 internal s_maxGasBufferToUpdateState;
```

Comment:
Let's make it immutable to save the setting functions and the gas for reading. That allows us to set it to a different value for chains with strange gas behaviour, and changing gas costs are extremely rare. With 1.7, swapping out a single offramp is possible so even if we need to deploy a new one it should be fine.